### PR TITLE
Term linking

### PR DESCRIPTION
**Pull request by henryiii**
This attempts to add linking syntax. This can be expanded to other Kits.

Renaming pages to uppercase letters.

### CODE REVIEW

**Comment by henryiii** [comment]
If it's helpful, the sed term is (in fish):
```bash
gsed -i 's/\[\([[:alnum:]]*\)\]([[:upper:]]\.[^)]*)/«\1»/g' **.md
```
And the rename:
```bash
rename -x -C *
rename -a.md *
```
(macOS requires more fiddling in the middle).

File: glossary/D.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Event selection and data analysis application. [Project website](lhcbdoc.web.cer
 
 ## DIRA: Direction angle {#DIRA}
 
-The angle between a line drawn from the [primary vertex](p.md#PV) to the decay vertex of the particle and the sum of the 4-momentum of its decay products.
+The angle between a line drawn from the [primary vertex](P.md#PV) to the decay vertex of the particle and the sum of the 4-momentum of its decay products.
```
Comment (eduardo-rodrigues) [inline glossary/D.md]: Seems this one link was not changed to the new format with a colon?

File: glossary/L.md [hunk 2]
```diff
@@ -35,16 +35,16 @@ Logical File Name. A path for a file stored on the grid that does not depend on
...
+An initial document submitted to the «LHCC» of CERN describing interest in performing a new experiment or major upgrade,
 outlining the physics case and detector required in broad terms.
-The [LHCC](l.md#LHCC) reviews the document and determines if the collaboration should proceed to the next stage of approval,
-typically a [Technical Proposal](t.md#TP) or Framework [TDR](t.md#TDR).
+The «LHCC» reviews the document and determines if the collaboration should proceed to the next stage of approval,
+typically a «Technical Proposal:TP» or Framework «TDR».
 
 For the original LHCb experiment this is [CERN/LHCC-95-5](http://cds.cern.ch/record/290868/files/SC00000024.pdf).
-For the LHCb Upgrade I there was initially a shorter [Expression of Interest](e.md#EoI) document
+For the LHCb Upgrade I there was initially a shorter [Expression of Interest](E.md#EoI) document
```
Comment (eduardo-rodrigues) [inline glossary/L.md]: Seems this one link was not changed to the new format with a colon?

File: glossary/L.md [hunk 3]
```diff
@@ -35,16 +35,16 @@ Logical File Name. A path for a file stored on the grid that does not depend on
...
 [CERN/LHCC-2008-007](http://cds.cern.ch/record/1100545/files/lhcc-2008-007.pdf),
-and this was followed by a [Letter of Intent](l.md#LoI) document [CERN/LHCC-2011-001](http://cds.cern.ch/record/1333091/files/LHCC-I-018.pdf).
-For the LHCb Upgrade II the [Expression of Interest](e.md#EoI) is [CERN-LHCC-2018-003](http://cds.cern.ch/record/2244311/files/PII_EoI_final_v3.pdf),
+and this was followed by a [Letter of Intent](L.md#LoI) document [CERN/LHCC-2011-001](http://cds.cern.ch/record/1333091/files/LHCC-I-018.pdf).
```
Comment (eduardo-rodrigues) [inline glossary/L.md]: Seems this one link was not changed to the new format with a colon?

File: glossary/L.md [hunk 4]
```diff
@@ -35,16 +35,16 @@ Logical File Name. A path for a file stored on the grid that does not depend on
...
+For the LHCb Upgrade II the [Expression of Interest](E.md#EoI) is [CERN-LHCC-2018-003](http://cds.cern.ch/record/2244311/files/PII_EoI_final_v3.pdf),
```
Comment (eduardo-rodrigues) [inline glossary/L.md]: Seems this one link was not changed to the new format with a colon?

File: glossary/P.md [hunk 5]
```diff
@@ -35,15 +35,15 @@ Advisory body to the LHCb management that defines the strategy and priority of t
...
+A request for processing centrally a physics request. Production Requests are managed by the LHCb distributed computing team via LHCbDIRAC. LHCb users can create production requests via the [LHCbDIRAC Web Portal](https://lhcb-portal-dirac.cern.ch/DIRAC/?view=tabs&theme=Grey&url_state=1|*LHCbDIRAC.ProductionRequestManager.classes.ProductionRequestManager). Production Requests are subject to approval by the «PPG».
 
 ## Productions {#Productions}
 
-Centrally managed processing, via LHCbDIRAC, of Production Requests (p.md#PR). Examples of productions include: MC simulation, Reconstruction, Stripping, Working Group productions. A production is a combination of application steps.
+Centrally managed processing, via LHCbDIRAC, of Production Requests (P.md#PR). Examples of productions include: MC simulation, Reconstruction, Stripping, Working Group productions. A production is a combination of application steps.
```
Comment (eduardo-rodrigues) [inline glossary/P.md]: Issue with the link itself.

File: glossary/T.md [hunk 6]
```diff
@@ -3,11 +3,11 @@
 ## T&A: Tracking and Alignment (and Vertexing) {#TandA}
 
 LHCb physics performance working group for matters of Tracking, Alignment, and Vertexing.
-Also represented in the [OPG](o.md#OPG) as Alignment and as Tracking.
+Also represented in the «OPG» as Alignment and as Tracking.
 
 ## Tagging
 
-See [Flavour Tagging](f.md#flavourtagging).
+See [Flavour Tagging](F.md#flavourtagging).
```
Comment (eduardo-rodrigues) [inline glossary/T.md]: Seems this one link was not changed to the new format with a colon?

File: glossary/D.md (hunk 1 above)
Comment (henryiii) [reply glossary/D.md]:
```suggestion
The angle between a line drawn from the «primary vertex:PV» to the decay vertex of the particle and the sum of the 4-momentum of its decay products.
```

File: glossary/L.md (hunk 2 above)
Comment (henryiii) [reply glossary/L.md]:
```suggestion
For the LHCb Upgrade I there was initially a shorter «Expression of Interest:EoI» document
```

File: glossary/L.md (hunk 3 above)
Comment (henryiii) [reply glossary/L.md]:
```suggestion
and this was followed by a «Letter of Intent:LoI» document [CERN/LHCC-2011-001](http://cds.cern.ch/record/1333091/files/LHCC-I-018.pdf).
```

File: glossary/L.md (hunk 4 above)
Comment (henryiii) [reply glossary/L.md]:
```suggestion
For the LHCb Upgrade II the «Expression of Interest:EoI» is [CERN-LHCC-2018-003](http://cds.cern.ch/record/2244311/files/PII_EoI_final_v3.pdf),
```

File: glossary/P.md (hunk 5 above)
Comment (henryiii) [reply glossary/P.md]:
```suggestion
Centrally managed processing, via LHCbDIRAC, of «Production Requests:PR». Examples of productions include: MC simulation, Reconstruction, Stripping, Working Group productions. A production is a combination of application steps.
```

File: glossary/T.md (hunk 6 above)
Comment (henryiii) [reply glossary/T.md]:
```suggestion
See «Flavour Tagging:Flavour-tagging».
```
This needs to be corrected by removing the `#flavourtagging` on the definition.

File: glossary/D.md (hunk 1 above)
Comment (eduardo-rodrigues) [reply glossary/D.md]: Looks good. I can't see how I can accept your suggested change automatically, though.

**Comment by henryiii** [comment]
I batched them and accepted them in one go. You need to be in the code tab to accept multiple changes, though both tabs should allow you to accept one at a time. If you want to make a suggestion, it's a little button at the top of the edit box when commenting on code.

We'll see what that looks like rendered...

**Comment by henryiii** [comment]
And, as you noticed, my search and replace missed terms with spaces (intentionally since those were harder to convert to the shortcut form).

**Comment by henryiii** [comment]
And, to type `«` and `»`, it's `Option \` and `Option Shift \` on a Mac. (`⌥\` and `⌥⇧\`). Alt 174 and Alt 175 on Windows. Or copy and paste, which is what I did.

**Pull request merged**